### PR TITLE
Updated to Swift 4.

### PR DESCRIPTION
**Pull request by elland**
Changes were quite minimal. Tagged to `1.2`.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+jobs:
+    build-and-test:
+        macos:
+            xcode: "9.1"
+        steps:
+            - checkout
+            - run:
+                name: Run tests
+                command: set -o pipefail && xcodebuild clean build test -workspace Toshi.xcworkspace -scheme "Tests" -destination 'platform=iOS Simulator,name=iPhone 7' | xcpretty --color && exit ${PIPESTATUS[0]}
```
Comment (designatednerd) [inline .circleci/config.yml]: did you mean for this to be toshi instead of teapot?

Comment (elland) [reply .circleci/config.yml]: Neither 🧠☠️

**Comment by elland** [comment]
@designatednerd alright, I think I got it now. :)

**Review by designatednerd — COMMENTED** [review]
🎉

**Comment by elland** [comment]
✨ 9.1.0 works!

**Pull request merged**